src/microsim/schema/simulation.py

```diff
@@ -108,6 +124,8 @@ def run(self, channels: int | Sequence[int] | None = None) -> xr.DataArray:
             channels = (channels,)
         images = []
         for channel_idx in channels:
+            self.illumination_flux(truth, channel_idx=channel_idx)
```

Comment:
i'm actually fine with merging this PR without `illumination_flux` actually being used anywhere yet.  I feel like i need some time to ponder some higher level overarching workflow stuff.  Maybe just remove this line for now